package.json

```diff
@@ -187,8 +190,12 @@
     "ts-loader": "^7.0.4",
     "tsc-alias": "^1.3.7",
     "typescript": "^4.6.3",
+    "vite": "^3.2.4",
+    "vite-plugin-stylelint": "^3.0.9",
+    "vite-plugin-vue2": "^2.0.2",
```

Comment:
`@vitejs/plugin-vue2` already exists, so I think you can remove this duplicated plugin, `vite-plugin-vue2`

Reply:
Oh thanx!